plugins/arclite/archive.hpp

```diff
@@ -169,7 +169,7 @@ class ArcAPI {
   static const ArcCodecs& codecs() {
     return get()->arc_codecs;
   }
-  static size_t Count7zCodecs() {
+  static const size_t Count7zCodecs() {
```

Comment:
const is useless on return types.

Reply:
fix it here https://github.com/FarGroup/FarManager/pull/723/commits/d832463aa807c429ca15719bcc7cb5939f306fbd